apps/webstore/src/app/pages/product/product.component.ts

```diff
@@ -399,4 +416,17 @@ export class ProductComponent implements OnInit {
       navigator.platform
     );
   }
+
+  /**
+   * Toggles between whether or not the box options like drag and close buttons should be visible.
+   * Changes the icon depending on the state.
+   */
+  toggleBoxOptions(): void {
+    this.showOptionBoxButtons = !this.showOptionBoxButtons;
+    if (this.showOptionBoxButtons) {
```

Comment:
This if statement and enum is unnecessary since the icon can only have two states: active / disabled. It would be cleaner with an ng class:

Something like this:  

[ngClass] = "{showOptionBoxButton ?  'boxOptionVisible' :  'boxOptionHidden' }"
.boxOptionVisible { background: url('link/to/icon') } 
.boxOptionHidden { background: url('link/to/icon') }

Reply:
We have decided to keep the working code